web-services/deploy/application/src/main/wildfly/add-datawave-configuration.cli

```diff
@@ -14,11 +14,8 @@ module add --name=com.mysql.driver --dependencies=javax.api,javax.transaction.ap
 # Remove the Elytron security subsystem entirely. Our JAAS security setup doesn't work
 # with it due to some bugs (the RunAsLoginModule doesn't work -- WFLY-11892) so we'll
 # just continue to use the deprecated system for now
```

Comment:
If this older comment is no longer valid, then should be removed. Unless there's more verification testing needed(?)

Reply:
Sorry if I've jumped the gun on reviewing this PR. I was just really curious :-)

Reply:
Removed the comment and the remove subsystem lines. Elytron doesn't appear to interfere with out security as in previous versions.  Also, found out that if we do keep the remove elytron line, we also have to remove the jwt subsystem since that subsystem uses elytroon.